nuxeo-agenda-core/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -6,10 +6,5 @@ Bundle-Vendor: Nuxeo
 Bundle-Version: 1.0.0
 Nuxeo-Component: OSGI-INF/agenda-core-types-contrib.xml,
  OSGI-INF/agenda-ecm-types-contrib.xml,
- OSGI-INF/agenda-layout-contrib.xml,
  OSGI-INF/agenda-service-framework.xml,
- OSGI-INF/agenda-operations-contrib.xml,
- OSGI-INF/theme-contrib.xml,
- OSGI-INF/agenda-actions-contrib.xml,
- OSGI-INF/agenda-pageproviders-contrib.xml,
```

Comment:
You're right. I just checked `org.nuxeo.ecm.platform.query.api.PageProviderService`, the class to which component `agenda-pageproviders-contrib.xml` contributes. It 's a backend service providing content to each page of Agenda using NXQL.